app/views/notification_mailer/notification_email.text.erb

```diff
@@ -1,11 +1,10 @@
 Nightingale Notification, <%= @user.email %>
 ===============================================
-There is a new Nightingale notification!
-You have a new Death Record to view: <%= link_to "Edit", edit_death_record_url(@death_record) %>.
+You have Death Records that require your attention!
 <% if @comments.any? %>
   Comments from previous user(s):
   <% @comments.each do |comment| %>
     <%= comment %>
```

Comment:
The place where I found the comments most confusing was in the email I received. Could we include the to/from information for each comment? More specifically: I was playing the role of registrar, and the email included the text "Comments from previous user(s): Sending this back to you to re-send back to me...", which was a bit confusing.

Reply:
I modified comments that arise from 'requesting changes' a bit above in the Step component, so they looks something like:
```
doc1@example.org requested edits from fd1@example.com. Comment: Please fill in the social security number, as it is missing.
```
Since this block essentially prints every comment currently on the record (including ones that didn't come from requesting edits), it might be trickier doing that here than how I did it above? Thoughts?

Reply:
Perhaps remove all comments except ones that are edit request comments directed at the recipient?